Sourcery.xcodeproj/project.pbxproj

```diff
@@ -1705,7 +1706,7 @@
 				CLANG_ENABLE_OBJC_WEAK = YES;
 				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
-				CODE_SIGN_IDENTITY = "Mac Developer";
+				CODE_SIGN_IDENTITY = "";
```

Comment:
Is this change necessary? Because this may break the release process.
If this is necessary, I'll see if this breaks during the next release.

Reply:
Not sure, this was changed by Xcode, however when I tried reverting this change, this is the warning I see:

```
TryCatch will not be code signed because its settings don't specify a development team. (in target 'TryCatch')
```

I would roll with Xcode recommendation, but up to you, tell me what you think, and I'll add a change with next commit.